tests/examples/example_SHR-2.out

```diff
@@ -4,235 +4,220 @@
-      Number of production wells:                             2
-      Number of injection wells:                              2
-      Flowrate per production well:                          55.0 kg/sec
-      Well depth (or total length, if not vertical):          7.5 kilometer
-      Geothermal gradient:                                    0.0500 degC/m
-
+      End-Use Option                               : Electricity
+      Average Net Electricity Production           :      30.58 MW
+      Electricity breakeven price                  :       3.92 cents/kWh
+      Number of production wells                   :          2
+      Number of injection wells                    :          2
+      Flowrate per production well                 :       55.0 kg/sec
+      Well depth (or total length, if not vertical):        7.5 kilometer
+      Geothermal gradient                          :     0.0500 °C/m
 
                            ***ECONOMIC PARAMETERS***
 
-      Economic Model = Fixed Charge Rate (FCR)
-      Fixed Charge Rate (FCR):                                5.00
-      Accrued financing during construction:                  0.00
-      Project lifetime:                                      30 yr
-      Capacity factor:                                       90.0 %
-      Project NPV:                                         115.21 MUSD
-      Project IRR:                                          11.74 %
-      Project VIR=PI=PIR:                                    1.72
-      Project MOIC:                                          2.42
-      Project Payback Period:                                9.54 yr
-
-                          ***ENGINEERING PARAMETERS***
-
-      Number of Production Wells:                             2
-      Number of Injection Wells:                              2
-      Well depth (or total length, if not vertical):          7.5 kilometer
-      Water loss rate:                                        2.0
-      Pump efficiency:                                       80.0
-      Injection temperature:                                 50.0 degC
-      Production Wellbore heat transmission calculated with Ramey's model
-      Average production well temperature drop:              19.0 degC
-      Flowrate per production well:                          55.0 kg/sec
-      Injection well casing ID:                               7.000 in
-      Production well casing ID:                              7.000 in
-      Number of times redrilling:                             0
-      Power plant type:                                       Single-Flash
-
-
-                         ***RESOURCE CHARACTERISTICS***
-
-      Maximum reservoir temperature:                        500.0 degC
-      Number of segments:                                     1
-      Geothermal gradient:                                    0.0500 degC/m
-
+      Economic Model                               : Fixed Charge Rate (FCR)
```

Comment:
What is the motivation for aligning the colons like this instead of the standard/regular way they currently? I don't believe I've ever seen this style of textual representation for data.